internal/ignition/ignition.go

```diff
@@ -591,6 +597,41 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte,
 	return nil
 }
 
+func (g *installerGenerator) importClusterTLSCerts() error {
+	tlsDir := os.Getenv("ASSISTED_SERVICE_CLUSTER_TLS_CERTS_DIR")
+	if tlsDir == "" {
+		return nil
+	}
+	entries, err := os.ReadDir(tlsDir)
```

Comment:
Please add logging if tlsDir, if it exists, and for each file copied to the tls dir.